frontend/src/components/UserCard.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { Button } from "semantic-ui-react";
+import { useAuthContext } from "../contexts/AuthContext";
+
+const UserCard = () => {
+  const { user, signOut } = useAuthContext();
+
+  // css styles
+  const userCardStyles = `container flex flex-col p-6 mx-auto justify-center items-center 
+    space-y-4 rounded-xl`;
+  const buttonStyles = ``;
+
+  const handleSignOut = async () => {
+    console.log("Signing out...");
+    const result = await signOut();
+    console.log(result);
+  };
+  return (
+    <div className={`${userCardStyles}`} id="user-card">
```

Comment:
Why not just className={userCardStyles}?

Reply:
I initially had it like that in case I wanted to add another css class that was not in Tailwind for some of the Semantic components. I will change it back until the need arises in the future.